_layouts/index.html

```diff
@@ -1,8 +1,31 @@
 ---
 layout: default
 ---
+
+<!-- Hero -->
+
+<div class="krx-hero">
+    <div class="container col-xxl-8 px-4 py-5">
+    <div class="row flex-lg-row-reverse align-items-center g-5 py-5">
+      <div class="col-10 col-sm-8 col-lg-6">
+        <img src="{{site.baseurl}}/images/kroxylicious_logo_rgb_no_text.svg" class="d-block mx-lg-auto img-fluid" alt="Kroxylicious Mascot" width="700" height="500" loading="lazy">
+      </div>
+      <div class="col-lg-6">
+        <h1 class="display-5 fw-bold text-body-emphasis lh-1 mb-3">Focus on the logic, not minutae.</h1>
```

Comment:
As @k-wall says I like the tag line but its not quite right. 

The only tag line I've come up with so far is 

> Kroxylicious augmenting the Kafka protocol since 2022.

Which while I like the sound of doesn't fit the hero usage either.